src/main/java/in/ashwanthkumar/gocd/slack/ruleset/Rules.java

```diff
@@ -214,6 +224,11 @@ public static Rules fromConfig(Config config) {
             password = config.getString("password");
         }
 
+        boolean displayConsoleLogs = true;
+        if (config.hasPath("display-console-logs")) {
+            displayConsoleLogs = config.getBoolean("display-console-logs");
```

Comment:
@jdavisp3 Can you please rename this to `display-console-log-links` since there's an another issue like #54 which wants to display the actual logs in the notification. I just want to call out the differences in case we choose do that as well in the future.